habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -408,7 +409,23 @@ def _load_navmesh(self, ep_info):
             base_dir = osp.join(*ep_info.scene_id.split("/")[:2])
 
         navmesh_path = osp.join(base_dir, "navmeshes", scene_name + ".navmesh")
-        self.pathfinder.load_nav_mesh(navmesh_path)
+        load_success = self.pathfinder.load_nav_mesh(navmesh_path)
+
+        # If we cannot load the navmesh, recompute the navmesh and cache it.
+        if not load_success:
+            navmesh_settings = NavMeshSettings()
+            navmesh_settings.set_defaults()
+            agent_config = self.get_agent(0).agent_config
+            navmesh_settings.agent_radius = agent_config.radius
+            navmesh_settings.agent_height = agent_config.height
+            navmesh_settings.agent_max_climb = 0.01
+            self.recompute_navmesh(
+                self.pathfinder,
+                navmesh_settings,
+                include_static_objects=True,
+            )
+            os.makedirs(osp.dirname(navmesh_path), exist_ok=True)
+            self.pathfinder.save_nav_mesh(navmesh_path)
```

Comment:
call `self.pathfinder.load_nav_mesh(navmesh_path) ` here to load the navmesh?

Reply:
Yes, it is.